crates/matrix-sdk/src/event_cache/mod.rs

```diff
@@ -954,7 +1032,9 @@ mod tests {
             let room_id = room_id!("!galette:saucisse.bzh");
             client.base_client().get_or_create_room(room_id, matrix_sdk_base::RoomState::Joined);
 
-            client.event_cache().subscribe().unwrap();
+            let event_cache = client.event_cache();
+
+            event_cache.subscribe().unwrap();
```

Comment:
Can you undo this change? The local isn't reused.

Reply:
I forgot to change something else, it's done.